tests/unit/CoreImageTest.php

```diff
@@ -41,98 +43,435 @@ public function tearDown(): void {
 		parent::tearDown();
 	}
 
-	public function test_retrieve_core_image_media_details() {
-		$query  = '
+	/**
+	 * Get the query for the CoreImage block.
+	 *
+	 * @param string $attributes The attributes to add to query.
+	 */
+	public function query( $attributes = '' ): string {
```

Comment:
This new param isnt in use, lets remove it.

(If it was in use, there'd be a longer discussion whether the ingenuity here outweights a general desire to not obfuscate whats getting tested. Either way, smart approach 🙌)

Reply:
Oh yes, i was actually using it to return a valid query for every untested attribute ( ) check by just passing that label to it. but later i thought to instead use the query in each case to make it more readable